api/api/util.py

```diff
@@ -252,11 +253,13 @@ def _create_problem(exc: Exception, code=None):
     ------
         ProblemException or `exc` exception type
     """
+    ext = None
```

Comment:
Not neccesary, all exceptions using ext are WazuhExceptions after removing APIError from here